cmd/kubeadm/app/cmd/alpha/kubeconfig_test.go

```diff
@@ -37,19 +37,19 @@ import (
 
 func generateTestKubeadmConfig(dir, id, certDir, clusterName string) (string, error) {
 	cfgPath := filepath.Join(dir, id)
-	initCfg := kubeadmapiv1beta2.InitConfiguration{
+	initCfg := kubeadmapiv1.InitConfiguration{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "kubeadm.k8s.io/v1beta2",
+			APIVersion: "kubeadm.k8s.io/v1beta3",
```

Comment:
Not for this PR, but we should really make all those constants (here and in other files) to use `kubeadmapiv1.SchemeGroupVersion.String` so the next release bump will be simpler.
@neolit123 opinions?
@RA489 would you like to take a shot at this?